src/lib/alerts.ts

```
@@ -0,0 +1,10 @@
+export const getAlert = async (): Promise<string | undefined> => {
+  return
+  try {
+    const response = await fetch('https://raw.githubusercontent.com/arkade-os/wallet/riga/motd.html')
+    const message = await response.text()
+    return message
+  } catch (error) {
+    console.error('Error fetching alert:', error)
+  }
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)